conda/environments/all_cuda-118_arch-x86_64.yaml

```diff
@@ -14,6 +14,7 @@ dependencies:
 - cudatoolkit
 - cudf==24.10.*,>=0.0.0a0
 - cupy>=12.0.0
+- cuvs==24.10.*,>=0.0.0a0
```

Comment:
I think you are correct here (we only need `cuvs` for the requirements/pyproject outputs for wheels). Let's file a follow-up PR once this one merges so we don't lose track. It can target 24.12.

Thankfully, `cuvs` is only ~350 kB: https://anaconda.org/rapidsai-nightly/cuvs/files